src/pages/Registration/Steps/Documentation/Form/index.tsx

```diff
@@ -77,7 +77,7 @@ export default function Form() {
         fieldName="hqCountry"
         placeholder="Select a country"
         classes={{
-          container: "px-4 bg-orange-l6 dark:bg-blue-d7",
+          container: "px-4",
```

Comment:
remove this background so that country selector is consistent with other field backgrounds